example/lambdas/python-reference-activity/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@cumulus/python-reference-activity",
   "private": true,
-  "version": "1.19.0",
+  "version": "1.21.0",
   "description": "Python reference activity",
-  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/python-reference-activity",
+  "homepage": "https://github.com/nasa/cumulus/tree/master/example/lambdas/python-reference-activity",
```

Comment:
Were these changes made in the release branch?

Reply:
No. I had to move them to make integration tests work with the released zips as they are not built and included in the release. They are just for our testing purposes. In accordance with moving them I updated this path. There is no change to our released packages or to anything except our `example` deployment.

Reply:
The two `python-reference` task packages were not published as they are set to `"private": true`.

Reply:
> made in the release branch?

If you mean `release-1.21.0`, then no. If you mean `release-1.21.x` then yes.